### PR TITLE
Add Chinese version of index page

### PR DESCRIPTION
**Pull request by nevermosby**
This PR is to add Chinese version of index page so that `ebpf.io` can support multi-language.

**It is WIP and will complete it before the end of this week.**

### CODE REVIEW

**Comment by nevermosby** [comment]
Hi, this PR has been done as I have translated most of index.js into Chinese. Pls help review it. Thx.

**Comment by joestringer** [comment]
:wave: @nevermosby , thanks for the submission!

Netlify seems to be having some trouble trying to render this PR:

```
10:38:15 PM:   Error: ./~/babel-loader/lib?{"plugins":["/opt/build/repo/~/gatsby/dist/utils/b  abel-plugin-extract-graphql.js","/opt/build/repo/~/babel-plugin-add-module-exp  orts/lib/index.js","/opt/build/repo/~/babel-plugin-transform-object-assign/lib  /index.js"],"presets":[["/opt/build/repo/~/babel-preset-env/lib/index.js",{"lo  ose":true,"uglify":true,"modules":"commonjs","targets":{"browsers":["> 1%","la  st 2 versions","IE >= 9"]},"exclude":["transform-regenerator","transform-es201  5-typeof-symbol"]}],"/opt/build/repo/~/babel-preset-stage-0/lib/index.js","/op  t/build/repo/~/babel-preset-react/lib/index.js"],"cacheDirectory":true}!./src/  pages/zh-CN/index.js
10:38:15 PM:   Module not found: Error: Cannot resolve 'file' or 'directory' ../stylesheets/i  ndex.scss in /opt/build/repo/src/pages/zh-CN
10:38:15 PM:   resolve file
10:38:15 PM:     /opt/build/repo/src/pages/stylesheets/index.scss.js doesn't exist
    /opt/build/repo/src/pages/stylesheets/index.scss.jsx doesn't exist
    /opt/build/repo/src/pages/stylesheets/index.scss doesn't exist
  resolve directory
10:38:15 PM:     /opt/build/repo/src/pages/stylesheets/index.scss doesn't exist (directory de  fault file)
    /opt/build/repo/src/pages/stylesheets/index.scss/package.json doesn't exist   (directory description file)
  [/opt/build/repo/src/pages/stylesheets/index.scss.js]
10:38:15 PM:   [/opt/build/repo/src/pages/stylesheets/index.scss.jsx]
10:38:15 PM:   [/opt/build/repo/src/pages/stylesheets/index.scss]
10:38:15 PM:    @ ./~/babel-loader/lib?{"plugins":["/opt/build/repo/~/gatsby/dist/utils/babel  -plugin-extract-graphql.js","/opt/build/repo/~/babel-plugin-add-module-exports  /lib/index.js","/opt/build/repo/~/babel-plugin-transform-object-assign/lib/ind  ex.js"],"presets":[["/opt/build/repo/~/babel-preset-env/lib/index.js",{"loose"  :true,"uglify":true,"modules":"commonjs","targets":{"browsers":["> 1%","last 2   versions","IE >= 9"]},"exclude":["transform-regenerator","transform-es2015-ty  peof-symbol"]}],"/opt/build/repo/~/babel-preset-stage-0/lib/index.js","/opt/bu  ild/repo/~/babel-preset-react/lib/index.js"],"cacheDirectory":true}!./src/page  s/zh-CN/index.js 23:0-36
```

I think we need to figure out why this is and resolve it first. Were you able to build & view the changes locally?

File: src/pages/zh-CN/index.js [hunk 1]
```diff
@@ -0,0 +1,330 @@
+import React from "react";
+import Link from "gatsby-link";
+import { unescape, shuffle } from "lodash";
+import { format, parse as parseDate } from "date-fns";
+import Slider from "infinite-react-carousel";
+
+import "../stylesheets/index.scss";
```
Comment (genbit) [inline src/pages/zh-CN/index.js:7]: this need to be 
```
import "../../stylesheets/index.scss";
```

File: src/pages/zh-CN/index.js [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+`;
+
+const monitoringText = `
+与依赖于操作系统提供的静态计数器和计量器不同，eBPF支持自定义指标的收集和内核聚合的能力，并基于广泛的可能来源生成可视化事件。
+通过只收集所需的可视化数据，在事件来源处生成直方图和类似的数据结构，而不是依赖样本的导出，这样可以扩展能够实现的可视化深度，并显著降低系统的总体开销。
+`;
+
+const Title = () => (
+  <hgroup>
+    <img className="main-logo" src={require("../assets/logo-big.png")} />
```
Comment (genbit) [inline src/pages/zh-CN/index.js:33]: needs to be 
```
    <img className="main-logo" src={require("../../assets/logo-big.png")} />
```

File: src/pages/zh-CN/index.js [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+  <div className="intro">
+    <p>
+      Linux内核一直是实现监控/可观察性、网络和安全的理想环境。
+      不幸的是，这往往是不切实际的，因为它需要改变内核源代码或加载内核模块，并导致生成互相层叠在一起的抽象层。
+      eBPF是一项革命性的技术，它可以在不更改内核源代码或不加载内核模块的情况下，在Linux内核中运行沙盒程序。
+    </p>
+    <p>
+      为了实现Linux内核的可编程性，基础架构软件可以利用现有的层次，使它们更加智能、功能更加丰富，而不是继续给系统增加额外的复杂度或降低执行效率和安全性。
+    </p>
+    <img src={require("../assets/go.png")} />
```
Comment (genbit) [inline src/pages/zh-CN/index.js:66]:
```
    <img src={require("../../assets/go.png")} />
```

File: src/pages/zh-CN/index.js [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+    );
+  }
+}
+
+const Sections = () => (
+  <div className="main-sections" style={{ marginTop: "30px" }}>
+    <div className="main-sections-left">
+      <Section
+        title="安全"
+        icon={require("../assets/intro_security.png")}
```
Comment (genbit) [inline src/pages/zh-CN/index.js:136]:
```
        icon={require("../../assets/intro_security.png")}
```

File: src/pages/zh-CN/index.js [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+        text={securityText}
+      />
+      <Section
+        title="网络"
+        icon={require("../assets/intro_networking.png")}
```
Comment (genbit) [inline src/pages/zh-CN/index.js:141]:
```
        icon={require("../../assets/intro_networking.png")}
```

File: src/pages/zh-CN/index.js [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+        text={networkingText}
+      />
+    </div>
+    <div className="main-sections-right">
+      <Section
+        title="追踪 & 分析"
+        icon={require("../assets/intro_tracing.png")}
```
Comment (genbit) [inline src/pages/zh-CN/index.js:148]:
```
        icon={require("../../assets/intro_tracing.png")}
```

File: src/pages/zh-CN/index.js [hunk 7]
```diff
@@ -0,0 +1,330 @@
...
+        text={tracingText}
+      />
+      <Section
+        style={{ marginTop: "45px" }}
+        title="观测 & 监控"
+        icon={require("../assets/intro_observability.png")}
```
Comment (genbit) [inline src/pages/zh-CN/index.js:154]:
```
        icon={require("../../assets/intro_observability.png")}
```

**Review by genbit — CHANGES_REQUESTED** [review]
need to fix relative paths

**Comment by nevermosby** [comment]
@genbit thx for remind and I have fixed the wrong code.

@joestringer thx for review and pls help review again.

**Review by joestringer — APPROVED** [review]
Text LGTM, though I couldn't figure out how to switch locales to observe the page in-place with the netlify preview.

**Comment by genbit** [comment]
> Text LGTM, though I couldn't figure out how to switch locales to observe the page in-place with the netlify preview.

@joestringer I don't think we have support for locale switching yet, we need to add support for it.

There are two options:

1. Add manual switching only - flags that basically goes to different pages
2. Add automatic locale detection based on IP and redirect to appropriate page. Netlify supports it - https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-country-or-language

**Comment by genbit** [comment]
@joestringer if you just want to look at the page: https://deploy-preview-8--ebpfio.netlify.app/zh-CN/

**Comment by joestringer** [comment]
@genbit do you want @nevermosby to take a crack at adding that support before merging this PR or are you happy to merge this as-is then someone can add locale switching afterwards?

**Comment by genbit** [comment]
@joestringer I'm ok with both as long as we track and have that someone to do redirect switching. If we close PR, we might just forget about it.

**Comment by qmonnet** [comment]
The Chinese translation is now merged via #96, and live on the website (see language selector at the top). Thanks a lot @nevermosby!

**Pull request closed without merging**